testing/poc/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: '3.9'
```

Comment:
Although docker compose is present in the testcontainers-go library, I'd enforce a practice of creating the services programmatically, where you can use Testcontainers APIs, such as waitFor (logs, ports, etc), mounts, etc. That would reduce the maintenance over those compose files, having the dependencies directly in the code, not in separate files.

All those compose files with just one service could be replaced with a GenericContainer from Testcontainers.